src/main/scala/rps/model/Request.scala

```diff
@@ -1,3 +1,3 @@
 package rps
 
-case class RPSRequest(userMove: String)
\ No newline at end of file
+case class Request(userMove: String)
```

Comment:
This doesn't need to be a `String`

Hint: this will cause you to rewrite and remove a bunch of code :P